# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: ipc/glue/BackgroundParent.h
# Commit: c8a1e52bfc49
# Full Hash: c8a1e52bfc49cb809bfcdfcd5f3635615579bc86
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2023-11-28 09:49:07
# Description:
#   Bug 1862723 - Use nsCString types for BackgroundParent::KillHardAsync, r=ipc-reviewers,mccr8
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D194548
# ==============================================================================

diff -r 008c188bf70b -r c8a1e52bfc49 ipc/glue/BackgroundParent.h
--- a/ipc/glue/BackgroundParent.h	Mon Nov 27 17:41:45 2023 +0000
+++ b/ipc/glue/BackgroundParent.h	Mon Nov 27 17:49:06 2023 +0000
@@ -78,7 +78,7 @@
   static uint64_t GetChildID(PBackgroundParent* aBackgroundActor);
 
   static void KillHardAsync(PBackgroundParent* aBackgroundActor,
-                            const char* aReason);
+                            const nsACString& aReason);
 
  private:
   // Only called by ContentParent for cross-process actors.